packages/jest-core/src/getChangedFilesPromise.ts

```diff
@@ -37,9 +37,6 @@ export default (
       console.error(chalk.red(`\n\n${message}`));
 
       process.exit(1);
-
-      // We do process.exit, so this is dead code
-      return Promise.reject(e);
```

Comment:
?

Reply:
This commit was made 2 years ago by someone. Code didn't seem to reach this point, so it was removed. Did I miss something here?

Reply:
unrelated to the other changes in this PR. Can you open up a PR to Jest removing this so it can land separately?

Reply:
@simenb: Should we undo this change from here?

Reply:
okay on it

Reply:
@sauravhiremath: I've pushed a branch for that